templates/zerver/help/change-your-settings.md

```diff
@@ -1,11 +1,12 @@
 # Change your settings
 
-1. Click the cog (<i class="icon-vector-cog"></i>) icon in the top right corner
-   of the right sidebar.
-2. From the dropdown menu, choose the **Settings** option.
-    <br>
-    ![settings-dropdown](/static/images/help/cog_dropdown_settings.png)
+Zulip has a wide variety of user settings that are available for changing.
 
-3. Click on the tabs on the left to select the settings you'd like to edit.
+1. From your desktop, click on the gear (<i class="icon-vector-cog"></i>) in
+the upper right corner.
 
-    ![settings](/static/images/help/display-settings.png)
+2. Select Settings.
+
+3. On the left, click the setting category that you would like to change.
+
+4. Within the main pane, edit the settings that you would like the change.
```

Comment:
I was thinking about adding links to all of the various settings help pages, in case people were searching for a specific setting and found the overall settings page instead, but then I thought that it would lead to bit-rot, because we are more likely to keep that index up to date in the sidebar rather than hidden in one of the help doc files.

Reply:
yeah, agreed. Possibly this page should just be instructions on how to get to https://chat.zulip.org/#settings/your-account, plus "click on the other tabs on the left to review and change your other settings."

Reply:
What do you think about calling the tabs "settings categories"? When I think of tabs, I think of tabs like browser tabs, imitating the tabs on paper folders in filing cabinets from the pre-computer dark ages.

Reply:
I just pushed an update to this doc, and I expanded a bit on your language:

If you can't find the setting that you would like to change, explore the other
settings categories on the left or explore the documentation for instructions
on the specific setting that you would like to change. If you still can't find
the setting, ask about it in the
[user community channel of the zulip development workspace](https://chat.zulip.org/#narrow/stream/138-user-community)

Reply:
FWIW - it doesn't look like Slack *has* a generic settings page, and this isn't actually doing much. Maybe we should just delete it? Aggressive pruning FTW?